packages/formspree-react/src/context.tsx

```diff
@@ -1,98 +1,75 @@
-import React, { Suspense, lazy, useContext, useEffect, useState } from 'react';
-import { createClient, getDefaultClient } from '@formspree/core';
-import type { Client, Config } from '@formspree/core';
+import { createClient, getDefaultClient, type Client } from '@formspree/core';
+import { Elements } from '@stripe/react-stripe-js';
 import { loadStripe } from '@stripe/stripe-js/pure.js';
-import type { Stripe } from '@stripe/stripe-js';
-
-const Elements = lazy(() =>
-  import('@stripe/react-stripe-js').then((module) => {
-    return { default: module.Elements };
-  })
-);
```

Comment:
This doesn't regress lazy loading compare to `main` (see https://react-demo.formspree.io/), since we import `@stripe/react-stripe-js` in `useForm` synchronously.

We can try to make lazy loading work for `@stripe/react-stripe-js` but let's achieve it via Stripe as a separate dependency work instead.